model-optimizer/unit_tests/extensions/ops/select_test.py

```diff
@@ -164,15 +214,15 @@ def test_select_infer_condition_with_value(self, condition_shape, else_data_shap
         """
         condition_value = condition_value(condition_shape)
         else_value = else_value(else_data_shape)
-        than_value = than_value(than_data_shape)
+        then_value = then_value(then_data_shape)
         output_value = output_value(select_output_shape)
         graph = build_graph_with_attrs(nodes_with_attrs=self.nodes, edges_with_attrs=self.edges,
                                        update_nodes_attributes=[('condition_data', {'shape': np.array(condition_shape),
                                                                                     'value': condition_value}),
                                                                 ('else_data', {'shape': np.array(else_data_shape),
                                                                                'value': else_value}),
-                                                                ('than_data', {'shape': np.array(than_data_shape),
-                                                                               'value': than_value}),
+                                                                ('then_data', {'shape': np.array(then_data_shape),
+                                                                               'value': then_value}),
```

Comment:
the same

Reply:
have completely rewritten these tests, and types now are defined explicitly